docs/xaml-tools/creating-a-ui-by-using-xaml-designer-in-visual-studio.md

```diff
@@ -27,6 +27,9 @@ For advanced users, you can even [customize the XAML Designer](https://github.co
 > [!NOTE]
 > Xamarin.Forms does not support a XAML designer. To view your Xamarin.Forms XAML UIs and edit them while the app is running, use XAML Hot Reload for Xamarin.Forms. For more information, see the [XAML Hot Reload for Xamarin.Forms (Preview)](/xamarin/xamarin-forms/xaml/hot-reload/) page.
 
+> [!NOTE]
+> WinUI 3 / .NET MAUI XAML designer is currently under development and does not support at now, to view your WinUI 3 / .NET MAUI XAML UIs and edit them while the app is running, use XAML Hot Reload for WinUI 3 / .NET MAUI. For more information, see the [XAML Hot Reload](../xaml-tools/xaml-hot-reload.md) page.
```

Comment:
```suggestion
> WinUI 3 / .NET MAUI XAML designer is currently under development and is not supported in Visual Studio 2022. To view your WinUI 3 / .NET MAUI XAML UIs and edit them while the app is running, use XAML Hot Reload for WinUI 3 / .NET MAUI. For more information, see the [XAML Hot Reload](../xaml-tools/xaml-hot-reload.md) page.
```

Reply:
@maddymontaquila Please check my changes and review / approve the PR.

Reply:
Only note is that it is **not** currently under development - so as long as you can clarify that everything else is good! Thank you!!

Reply:
Revising the text to avoid saying it's under development.